raiden-ts/src/raiden.ts

```diff
@@ -475,14 +512,9 @@ export class Raiden {
    *
    * @param token  Token address on currently configured token network registry
    * @param partner  Partner address
-   * @param settleTimeout  openChannel parameter, defaults to 500
    * @returns  txHash of channelOpen call, iff it succeeded
    */
-  public async openChannel(
-    token: string,
-    partner: string,
-    settleTimeout: number = 500,
-  ): Promise<Hash> {
+  public async openChannel(token: string, partner: string): Promise<Hash> {
```

Comment:
Don't remove the `settleTimeout` parameter, one may want to specify a different than default at `openChannel` time. Instead, make it optional, or even better, in an `opts` optional parameter object for the case we want to add other options here in the future (as done in `transfer`):
```suggestion
  public async openChannel(token: string, partner: string, opts?: { settleTimeout?: number; }): Promise<Hash> {
```